packages/data-point/src/DataPoint.test.js

```diff
@@ -0,0 +1,198 @@
+const dataPoint = require("./DataPoint");
+const { Accumulator } = require("./Accumulator");
+const { Cache } = require("./Cache");
+
+const sayHello = value => `Hello ${value}`;
+const getAccumulator = (input, acc) => acc;
+
+const shallowTracer = {
+  startSpan: () => Object.create(shallowTracer),
```

Comment:
because we are only testing the API, `startSpan` creates a new object of type Span, so when I use this method inside:

https://github.com/ViacomInc/data-point/blob/61a6dc3d217647815b046170df8e7313b51441c8/packages/data-point/src/trace-span.js#L11-L13

it will expect the tracer to create a new span object.

Reply:
to expand on the answer, Object.create will create a copy of the `shawllowTracer` which now that I think of it, Im not even testing this funcitonality so ... prob I can remove it, going to try and if it passes then I will make a commit with the change

Reply:
so, im now returning just the same object since we are not checking if a new span is created or not